### DISCUSSION TITLE
Wallpaper Playback True If Desktop Is Visible.

### DISCUSSION THREAD

**Discussion by ShashTheEpic** [rocksdanister/lively#1611] [Ideas]
You should make a feature where if the desktop is visible wallpaper playback is true. Even if half of the screen is covered by an application, it would still play. But if a whole application is covering the screen, the wallpaper stops.

![Screenshot 2023-03-06 161503](https://user-images.githubusercontent.com/121744832/223027615-cd72d092-a61d-4e19-97b7-08c4b0d5607b.jpg)

**Comment by rocksdanister** [comment]
>Wallpaper Playback True If Desktop Is Visible

Its already there, just set `Application focused -> Pause`

> Even if half of the screen is covered by an application, it would still play. But if a whole application is covering the screen, the wallpaper stops.

This is the default behaviour unless you have some overlay application on desktop.

**Reply by ShashTheEpic** [reply]
I am requesting so that even if an application is focused, and the desktop is visible the wallpaper will be playing. So basically if the desktop is ever visible the wallpaper will be playing. Not so that when an application is open it pauses.

**Reply by rocksdanister** [reply]
This is the default behaviour unless you have some overlay application on desktop.

Wallpaper is paused when application covers 95% desktop area or fullscreen.

If its still something else then maybe try making an example screenshot showing the scenario.

**Reply by ShashTheEpik** [reply]
Ok thanks

**Comment by rocksdanister** [comment]
If I am misunderstanding what you are trying to say then maybe provide an example screenshot with a detailed description in feature request:
https://github.com/rocksdanister/lively/issues

**Reply by ShashTheEpic** [reply]
No need for that I just got confused
